.github/workflows/basic.yml

```diff
@@ -1,6 +1,6 @@
 name: BasicBuilds
```

Comment:
nit: This name could use some whitespace.

Reply:
If so, it should be changed as a separate, cosmetic change.